### PR TITLE
Includes custom options from feature into billboard generation

### PR DESCRIPTION
**Pull request by mikeydunnx**
Wondering if there would be any interest in a way to provide more flexibility in generating billboards through ol-cesium. This is more of a jumping off point from a coding perspective. My goal was to be able to provide cesium specific options for the billboard generation.

A feature could now include the key `bbOptions` where you could provide members available to the [billboards](https://cesium.com/docs/cesiumjs-ref-doc/Billboard.html).

```javascript
feature.bbOptions = {
  scaleByDistance: new Cesium.NearFarScalar(1.5e4, 1, 8.0e5, 0.0),
  pixelOffsetScaleByDistance: new Cesium.NearFarScalar(1.5e4, 1, 8.0e5, 0)
}
```

### CODE REVIEW

File: buildtools/webpack.commons.js [hunk 1]
```diff
@@ -9,13 +9,16 @@ const babelPresets = [['@babel/preset-env', {
...
+const babePlugins = ['@babel/plugin-transform-destructuring'];
+
 const olRule = {
   test: /ol\/.*\.js$/,
   use: {
     loader: 'babel-loader',
     options: {
       babelrc: false,
       presets: babelPresets,
+      plugins: babePlugins
```
Comment (brianpmaher) [inline buildtools/webpack.commons.js:21]: babe -> babel 😄

Comment (mikeydunnx) [reply buildtools/webpack.commons.js]: 🤦

File: src/olcs/FeatureConverter.js [hunk 2]
```diff
@@ -723,7 +723,8 @@ class FeatureConverter {
         color,
         scale,
         heightReference,
-        position
+        position,
+        ...feature.bbOptions
```
Comment (gberaudo) [inline src/olcs/FeatureConverter.js:727]: You can achieve the same without introducing destructuring with `Object.assign(bbOptions, feature.get('cesiumOptions'));`.

Comment (mikeydunnx) [reply src/olcs/FeatureConverter.js:727]: Yup, I'll make this update.

**Review by gberaudo — APPROVED** [review]

**Comment by gberaudo** [comment]
Thanks @mikeydunnx.

**Pull request merged**